release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2546,6 +2544,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042601[*BZ#2042601*])
+
+* If a `ClusterGroupUpgrade` CR validation fails for any reason, for example, because of an invalid spoke name, no status is available for the `ClusterGroupUpgrade` CR, as if the CR is disabled.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040828[*BZ#2040828*])
+
+* Currently, for `ClusterGroupUpgrade` CR state changes, there is only a single condition type available - `Ready`. The `Ready` condition can have a status of `True` or `False` only. This does not refect the range of states the `ClusterGroupUpgrade` CR can be in.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042596[*BZ#2042596*])
+
+* The alpha channel for the PTP Operator reports an error that prevents a `PtpConfig` CR from being applied when the `summary_interval` value does not match the value set for `logSyncInterval`. To work around this issue, update all reference PTP configurations to set `summary_interval` and `logSyncInterval` to -4.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040799[*BZ#2040799*])
+
+* When you create spoke clusters using policies and the OpenShift GitOps Operator v1.4.1 on a {rh-rhacm-first} hub cluster, clusters and policies apps report errors and do not recover.
++
+To work around this issue, pin the OpenShift GitOps Operator to v1.3.2. To do this, make the following changes on the hub cluster:
++
+. Apply the following `Subscription` CR to the hub cluster:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-operators
+spec:
+  name: openshift-gitops-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: openshift-gitops-operator.v1.3.2
+  installPlanApproval: Manual
+----
++
+. Optional: Manually approve the `installplan` created for the subscription:
++
+[source,terminal]
+----
+$ oc get installplan -n openshift-operators
+----
++
+.Expected output
+[source,terminal]
+----
+NAME            CSV                                APPROVAL   APPROVED
+install-7wdfv   openshift-gitops-operator.v1.3.2   Manual     false
+----
++
+.. Edit the `installplan` that has CSV version `v.1.3.2`:
++
+[source,terminal]
+----
+$ oc edit installplan -n openshift-operators install-7wdfv
+----
++
+.. Set the `approved` field to `true`:
++
+[source,yaml]
+----
+spec:
+  approval: Manual
+  approved: true
+  clusterServiceVersionNames:
+  - openshift-gitops-operator.v1.3.2
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046775[*BZ#2046775*])
+
+* Sometimes, when provisioning spoke clusters, argoCD applications with 100 or more `Siteconfig` CRs can cause `OOMKilled` memory errors on the hub cluster, even when the `openshift-gitops-application-controller-0` is allocated 40GB memory.
```

Comment:
@hanqiuzh review/LGTM - RAN GA known issue.

Reply:
The issue doesn't not exist but the siteconfig & ArgoCD still shipped with RAN 4.10